### PR TITLE
OM-63 fix cluster size alert

### PR DESCRIPTION
**Pull request by hev**

### CODE REVIEW

**Comment by spkesan** [comment]
Maybe we would want to review all the alert rules here and the usage of labels? What do you think @hev ?
There is also an alert dashboard that depends on these rules I think.

**Comment by hev** [comment]
@spkesan agreed that we should review all alerts. We have that planned. This one has just been bothering me as it fires every time I do testing.

File: config/prometheus/aerospike_rules.yml [hunk 1]
```diff
@@ -58,12 +58,12 @@ groups:
         description: 'Clock has skewed for namespace {{ $labels.ns }} in node {{ $labels.instance }}'
         summary: Writes will be stopped by Aerospike
     - alert: ClusterSize
-      expr: aerospike_node_stats_cluster_size{job="aerospike"} < 3 # user has to replace 3 here with their desired cluster size.
+      expr: sum by(cluster_name) (aerospike_node_stats_cluster_size{job="aerospike"}) < 3 # user has to replace 3 here with their desired cluster size.
```
[the remark this answers is not in the document]
Comment (hev) [reply config/prometheus/aerospike_rules.yml:61]: @spkesan `min by ` will alway be 1 correct? perhaps I don't understand this metric but I see each node reporting it as 1 which means a `sum by (cluster_name)` seems to make sense here. 

Explore_-_Aerospike_Prometheus_-_Grafana
Cursor_and_Explore_-_Aerospike_Prometheus_-_Grafana

[the remark this answers is not in the document]
Comment (sunilvirus) [reply config/prometheus/aerospike_rules.yml:61]: @hev The nodes should nt be reporting 1. seems your nodes did not really form cluster. i am thinking original rule is good. It seems to be alerting you accurately. Did you check if your cluster form properly ?

[the remark this answers is not in the document]
Comment (hev) [reply config/prometheus/aerospike_rules.yml:61]: OK. good to know. I will close this PR.

**Pull request closed without merging**